### PR TITLE
Issue#9286

### PR DESCRIPTION
**Pull request by huanghao495430759**
Issue#9286

### CODE REVIEW

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi the issues of this PR are mentioned in #9378. Could you give them a review?

**Comment by tristaZero** [comment]
Hi @huanghao495430759 

Kindly ping here. Could you look at these reviews?

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/mix.en.md [hunk 1]
```diff
@@ -3,4 +3,98 @@ title = "Mixed Rules"
 weight = 6
 +++
 
-TODO
+The overlay between rule items in a mixed configuration is associated by the data source name and the table name.
```
Comment (tristaZero) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/mix.en.md:6]: Same with  #9378 here.

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/mix.en.md [hunk 2]
```diff
@@ -3,4 +3,98 @@ title = "Mixed Rules"
...
+EncryptTableRuleConfiguration encryptTableRuleConfig = new EncryptTableRuleConfiguration("t_user", Arrays.asList(columnConfigAes, columnConfigTest));
+// Data encrypt algorithm configuration
+Map<String, ShardingSphereAlgorithmConfiguration> encryptAlgorithmConfigs = new LinkedHashMap<>(2, 1);
+encryptAlgorithmConfigs.put("name_encryptor", new ShardingSphereAlgorithmConfiguration("AES", encryptProperties));
+encryptAlgorithmConfigs.put("pwd_encryptor", new ShardingSphereAlgorithmConfiguration("assistedTest", encryptProperties));
+EncryptRuleConfiguration encryptRuleConfiguration = new EncryptRuleConfiguration(Collections.singleton(encryptTableRuleConfig), encryptAlgorithmConfigs);
+
+/* Replica query rule configuration */
+ReplicaQueryDataSourceRuleConfiguration dataSourceConfiguration1 = new ReplicaQueryDataSourceRuleConfiguration("ds_0", "primary_ds0", Arrays.asList("primary_ds0_replica0", "primary_ds0_replica1"), null);
+ReplicaQueryDataSourceRuleConfiguration dataSourceConfiguration2 = new ReplicaQueryDataSourceRuleConfiguration("ds_1", "primary_ds0", Arrays.asList("primary_ds1_replica0", "primary_ds1_replica0"), null);
```
Comment (tristaZero) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/mix.en.md:84]: The `loadBalancerName` is null?

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi,

I can see your conscientiousness at this issue, great.

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/mix.en.md (hunk 2 above)
Comment (huanghao495430759) [reply docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/mix.en.md:84]: Oh ,it's my mistake,i've reviewed it.

**Review by tristaZero — APPROVED** [review]
Nice job.

**Pull request merged**